buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextStyles.kt

```diff
@@ -30,10 +30,15 @@ interface BpkTextStyles : List<BpkTextStyleModel>
 
 data class BpkTextStyleModel(
   val name: String,
-  val fontSize: String,
-  val fontWeight: String,
-  val lineHeight: String,
-  val letterSpacing: String?,
+  val fontSize: Token,
```

Comment:
with the xml tokens we sometimes need the name and sometimes the actual value, so had to change the parsing a little